src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDateTime;
+import java.util.Comparator;
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.MovieSession;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
+import mate.academy.service.OrderService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order order = new Order();
+        order.setTickets(shoppingCart.getTickets());
+        order.setUser(shoppingCart.getUser());
+        List<Ticket> tickets = shoppingCart.getTickets();
+        LocalDateTime lastDate = tickets.stream()
```

Comment:
orderDate - is DateTime when the order is created. MovieSession ShowTime is completely another thing.